### PR TITLE
Add SPDX header - batch 9

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* Resolves: SPDX header for more files

## Summary

slowly but steady we are getting there...

* files with copyright information: 10031 / 10972
* files with license information: 9054 / 10972

## TODO

- [x] green CI

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
Anybody who can tell me how to correctly reformat the code, mentioned in https://github.com/nextcloud/server/actions/runs/9291082951/job/25568820867?pr=45573 ?

File: apps/user_ldap/tests/AccessTest.php [hunk 1]
```diff
@@ -1,34 +1,8 @@
...
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License, version 3,
- * along with this program. If not, see <http://www.gnu.org/licenses/>
- *
+ * SPDX-FileCopyrightText: 2016 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-FileCopyrightText: 2016 ownCloud, Inc.
+ * SPDX-License-Identifier: AGPL-3.0-only
```
Comment (come-nc) [inline apps/user_ldap/tests/AccessTest.php:5]: How come `user_ldap` is under AGPLv3 while the rest is under AGPLv3+? Is it too late to change that?

**Review by skjnldsv — APPROVED** [review]

**Review by come-nc — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
Thanks a lot for the fixes @come-nc 💯

**Pull request merged**